src/main/java/seedu/fridget/Fridget.java

```diff
@@ -22,15 +22,18 @@ public Fridget() {
 
     public void run() {
         ui.printIntroduction();
+        Command c = new Command();
 
-        try {
-            String userInput = ui.readUserInput();
-            ui.printSeparatorLine();
-            Command c = parser.parseCommand(userInput);
-            c.execute(ui, parser, ingredientList);
-        } catch (FridgetException e) {
-            e.printStackTrace();
-        }
+        do {
+            try {
+                String userInput = ui.readUserInput();
+                ui.printSeparatorLine();
+                c = parser.parseCommand(userInput);
+                c.execute(ui, parser, ingredientList);
+            } catch (FridgetException e) {
+                e.printStackTrace();
+            }
+        } while (c.status());
```

Comment:
Can we rename this to something clearer? Something like `while(!c.exitStatus())` or `while(c.exitNotRequired())`?